guides/extensions/rest.md

```diff
@@ -31,7 +31,7 @@ You can do this directly on your machine.
 After installing and confirming that Libindy is installed, simply run:
```

Comment:
I guess @credo-ts/rest will be available as soon as we get the 0.5.0 release. In such case Libindy will not be needed, right?

Reply:
I'm working on updating the REST PR and then releasing a new version. Will update there accordingly